azure-pipelines-official.yml

```diff
@@ -1,3 +1,9 @@
+# Branches that trigger a build on commit
+trigger:
```

Comment:
So i guess this work on both pipelines? Do we want to run it on devdiv only fro now (is it even possible to use condition here)?

Reply:
If you've configured the new pipeline the same as the old, then you are already overriding this section.